test/debugger/stepout-test1.js

```diff
@@ -0,0 +1,26 @@
+/*
+Basic test for stepOut
+Nested function calls -- stepOut on a breakpoint at line 23 (return 4 in function d())
+should allow us to step back into c, then b, then a. 
+*/
+
+let a = function() {
```

Comment:
The indention in the code looks a bit weird. You may want to be consistent with testcase to use 2 spaces indention.

Reply:
What do you mean? This test file is 2-space indented, and so are the exists tests.

Reply:
If you look at the indention at line 8, 9, 13, 14 etc..., they all have much bigger indention than 2.